modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,166 @@ The properties for broker-wide throughput quota balancing are configured at the
+[,bash]
+----
+rpk cluster quotas describe --name client-id=consumer-1
+----
 
-The values of both `target_quota_byte_rate` and `target_fetch_quota_byte_rate` are throughput rate limits within a shard, in bytes per second.
+To set a throughput quota for a single client, use the `rpk cluster quotas alter` command. 
+
+[,bash]
+----
+rpk cluster quotas alter --add consumer_byte_rate=200000 --name client-id=consumer-1
+----
 
 === Group of clients throughput limit
 
-To set throughput quota for a group of clients, configure the cluster properties xref:reference:cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`] and xref:reference:cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`].
+Alternatively, you can view or configure throughput quotas for a group of clients based on a match on client ID prefix. The following example sets the `consumer_byte_rate` quota to a group of clients that have IDs starting with `consumer-`:
 
-* The `kafka_client_group_byte_rate_quota` property applies to producer clients. It sets a maximum throughput quota of traffic sent to Redpanda from each producer in the group named by the property.
-* The `kafka_client_group_fetch_byte_rate_quota` property applies to consumer clients. It sets a maximum throughput quota of traffic fetched from Redpanda by each consumer in the group named by the property.
+[,bash]
+----
+rpk cluster quotas alter --add consumer_byte_rate=200000 --name client-id-prefix=consumer-
+----
+
+NOTE: The client group that matches on `client-id-prefix` does not denote a Kafka consumer group. Instead, it matches all requests with a `client_id` field that has the configured prefix. The `client_id` field is typically a configurable parameter in Kafka libraries when you create a client.
 
-Both `kafka_client_group_byte_rate_quota` and `kafka_client_group_fetch_byte_rate_quota` have the configuration value fields:
+=== Default client throughput limit
 
-* `group_name`: a name for the group of clients.
-* `clients_prefix`: a client belonging to the group has this prefix in its `client_id` property.
-* `quota`: the maximum throughput rate of *each client in the group* in bytes per second.
+You can apply default throughput limits to clients. Redpanda applies the default limits if no quotas have been configured specific to a client ID or ID prefix.
+
+The example below sets a produce quota of 1GBps through the Kafka API, which means a 1GBps limit across all produce requests sent to a single Redpanda broker. 
+
+[,bash]
+----
+rpk cluster quotas alter --default client-id --add producer_byte_rate=1000000000
+----
```

Comment:
@kbatuigas we also added an 'import' capability (just last week) to the RPK quotas support, which allows you to bulk import from a file to make it easier to manage a larger number of quotas.

This should also be documented.  It's mentioned here: https://github.com/redpanda-data/devex/issues/22

@r-vasquez has the details on the syntax.  But one import example here would be nice.

Reply:
Hi @kbatuigas, the PR includes some examples: https://github.com/redpanda-data/redpanda/pull/21311

// With a full valid JSON
```
rpk cluster quotas import --source '{"quotas":[{"entity":[{"name":"foo","type":"client-id"}],"values":[{"key":"consumer_byte_rate","values":"12123123"}]},{"entity":[{"name":"foo-","type":"client-id-prefix"}],"values":[{"key":"producer_byte_rate","values":"12123123"},{"key":"consumer_byte_rate","values":"4444444"}]}]}'
```

Something important to document, is that we mention in the help text that `import` takes the output of `rpk cluster quotas describe --format <json/yaml>`, so `describe --format` acts like an `export` command.